src/bun.js/bindings/exports.zig

```diff
@@ -1941,6 +2015,35 @@ pub const ZigConsoleClient = struct {
                     writer.print(comptime Output.prettyFmt("<r><yellow>{s}n<r>", enable_ansi_colors), .{out_str});
                 },
                 .Double => {
+                    if (value.isObject()) {
```

Comment:
I think CellType or something like that exists already. It would be better for perf to consult that